__tests__/app.test.js

```diff
@@ -244,6 +244,30 @@ describe("PATCH /api/articles/:article_id", () => {
     });
 })
 
+describe.only("DELETE /api/comments/:comment_id", () => {
+    test("status 204, responds with 204 after deleting the comment that corresponds with the id passed in", () => {
+        return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+    });
+    test("status 404, responds with 404 if a valid id that is not currently in the data is used", () => {
+        return request(app)
+        .delete("/api/comments/133")
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("Endpoint Not Found")
```

Comment:
Would be nicer if this msg said 'comment not found' you could do that by altering your promise.reject in the model - isn't technically wrong though!